src/behaviors/dragPanBehavior.ts

```diff
@@ -0,0 +1,89 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Behavior {
+  export class DragPan {
+
+    private _dragInteraction: Interaction.Drag;
+    private _scale: Scale.AbstractQuantitative<any>;
+    private _leftBounds = [-Infinity, Infinity];
+    private _rightBounds = [-Infinity, Infinity];
+    private _verticalPan: boolean;
+
+    /**
+     * Creates a DragPan Behavior.
+     *
+     * The allows you to move around and zoom in on a plot, interactively. It
+     * does so by changing the xScale and yScales' domains repeatedly.
+     *
+     * @constructor
+     * @param {QuantitativeScale} [xScale] The X scale to update on panning/zooming.
+     * @param {QuantitativeScale} [yScale] The Y scale to update on panning/zooming.
+     */
+    constructor(scale: Scale.AbstractQuantitative<number>, isVertical: boolean) {
+      this._scale = scale;
+      this._dragInteraction = new Interaction.Drag();
+      this._setupInteraction(this._dragInteraction);
+      this._verticalPan = isVertical;
+    }
+
+    public getInteraction(): Interaction.Drag {
+      return this._dragInteraction;
+    }
+
+    public leftBounds(): number[];
+    public leftBounds(newBounds: number[]): Behavior.DragPan;
```

Comment:
To be more explicit, this is so users can enact the min/max of the left edge of the domain.